### PR TITLE
New Post Button and New Post Page

### PR DESCRIPTION
**Pull request by brianwmlo**
urls.py - new url for the new post page
views.py - new view for the new post page
nav.html- placeholder for the new post button
post-new.html - template for new post page

### CODE REVIEW

File: blogger/post/views.py [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
     def get(self, request):
         return self.render_to_response({})
+
+
+class NewPost(TemplateView):
+	
+	template_name = 'post/post-new.html'
+
+	def get(self, request):
+		return self.render_to_response({})
```
Comment (derekkwok) [inline blogger/post/views.py]: Make sure to only indent by 4 spaces.

File: templates/post/post-new.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <article>
+    	<h1>New Blog Post</h1>
+        <h2>Title</h2>
```
Comment (derekkwok) [inline templates/post/post-new.html]: Make sure indentation is consistent.

File: templates/base/nav.html [hunk 3]
```diff
@@ -1 +1 @@
-<nav>This is the nav.</nav>
+<a>New Post</a>
```
Comment (derekkwok) [inline templates/base/nav.html]: Keep the <nav> tag and put the <a> inside the <nav>.

File: blogger/post/views.py (hunk 1 above)
Comment (brianwmlo) [reply blogger/post/views.py]: eh its weird...it looks fine in my script
capture

File: templates/post/post-new.html (hunk 2 above)
Comment (brianwmlo) [reply templates/post/post-new.html]: hmm same as above too..
capture

File: blogger/post/views.py (hunk 1 above)
Comment (derekkwok) [reply blogger/post/views.py]: So in the bottom right of sublime text, you'll see "tab size: 4". Make sure you have "indent using spaces" checked.

File: templates/post/post-new.html (hunk 2 above)
Comment (derekkwok) [reply templates/post/post-new.html]: So in the bottom right of sublime text, you'll see "tab size: 4". Make sure you have "indent using spaces" checked.

File: templates/base/nav.html (hunk 3 above)
Comment (derekkwok) [reply templates/base/nav.html]: This should read be:

    <nav>
        <a>New Post</a>
    </nav>

**Pull request merged**